samples/typescript/21.react-media-template/src/components/MediaPlayerContainer.tsx

```diff
@@ -187,25 +194,37 @@ export const MediaPlayerContainer: FC<MediaPlayerContainerProps> = ({
     const flexItemStyles = getFlexItemStyles();
     const playerControlStyles = getPlayerControlStyles();
     const videoStyle = getVideoStyle();
+    const resizeReferenceStyles = getResizeReferenceStyles();
```

Comment:
not a big deal but this could be 
`const { root as resizeReferenceStyles } = getResizeReferenceStyles();`

Reply:
(Follows fluent convention)